ldk/javascript/src/utils/mapper.ts

```diff
@@ -0,0 +1,68 @@
+import { TextEncoder, TextDecoder } from 'text-encoding-shim';
+import { HTTPResponse, Socket } from '../network';
+
+export const mapToUint8Array = (data: ArrayBuffer): Uint8Array => new Uint8Array(data);
+
+export const mapToHttpResponse = (response: OliveHelps.HTTPResponse): HTTPResponse => ({
+  statusCode: response.statusCode,
+  body: new Uint8Array(response.body),
+  headers: response.headers,
+});
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const handleCaughtError = (reject: (reason?: any) => void, error: Error, type: string) => {
+  console.error(`Received error calling ${type}: ${error.message}`);
+  reject(error);
+};
+
+enum MessageType {
+  text = 1,
+  binary = 2,
+}
+
+const mapToBinaryData = (message: string | Uint8Array) =>
```

Comment:
Can you specify return types for each of these functions?

Reply:
done